src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -604,8 +610,9 @@ void OrbitMainWindow::CreateTabBarContextMenu(QTabWidget* tab_widget, int tab_in
 void OrbitMainWindow::UpdateCaptureStateDependentWidgets() {
   auto set_tab_enabled = [this](QWidget* widget, bool enabled) -> void {
     QTabWidget* tab_widget = FindParentTabWidget(widget);
-    ORBIT_CHECK(tab_widget != nullptr);
```

Comment:
Will you revert this change soon? Consider adding a comment + a TODO

Reply:
Added a todo; do you think a check here is better?  I wasn't sure